refinery/templates/core/all_files.html

```diff
@@ -0,0 +1,251 @@
+    handle-overflow" id="data-set-view">
+      <!-- DIFF: Remove "if data_set" -->
+      <div ui-view>TODO</div>
+    </div>
+  </div>
+</div>
+
+{% endblock %}
+
+{% block vendor_scripts %}
+  <script type="text/javascript" src="{% static "js/d3/fisheye.js" %}"></script>
+  <script type="text/javascript" src="{% static "js/underscore/underscore-min.js" %}"></script>
+  <script type="text/javascript" src="{% static "js/backbone/backbone-min.js" %}"></script>
+  <script type="text/javascript" src="{% static "js/backbone-marionette/backbone.marionette.js" %}"></script>
+  <script type="text/javascript" src="{% static "js/bootbox/bootbox.min.js" %}"></script>
+
+  <!-- adding spinner -->
+  <script language="javascript" type="text/javascript" src="{% static "js/spin/spin.min.js" %}"></script>
+
+  <script language="javascript" type="text/javascript">
+    var opts = {
+      lines: 13, // The number of lines to draw
+      length: 4, // The length of each line
+      width: 2, // The line thickness
+      radius: 6, // The radius of the inner circle
+      corners: 1, // Corner roundness (0..1)
+      rotate: 0, // The rotation offset
+      color: '#000', // #rgb or #rrggbb
+      speed: 1, // Rounds per second
+      trail: 60, // Afterglow percentage
+      shadow: false, // Whether to render a shadow
+      hwaccel: false, // Whether to use hardware acceleration
+      className: 'spinner', // The CSS class to assign to the spinner
+      zIndex: 2e9, // The z-index (defaults to 2000000000)
+      top: 'top', // Top position relative to parent in px
+      left: 'top' // Left position relative to parent in px
+    };
+  </script>
+
+{% endblock %}
+
+{% block script %}
+
+
+  <!-- Refinery libraries -->
+  <script type="text/javascript">
+    var REFINERY_BASE_URL = document.location.host;
+    var REFINERY_API_BASE_URL = "/api/v1/";
+    var REFINERY_SOLR_BASE_URL = "{{ REFINERY_SOLR_BASE_URL }}";
+    var externalAssayUuid = "{{ assay_uuid }}";
+    var externalStudyUuid = "{{ study_uuid }}";
+    var externalAssayId = "{{ assay_id }}";
+    var externalStudyId = "{{ study_id }}";
+    var dataSetUuid = "{{ data_set.uuid }}";
+    var analysisUuid = "{{ analysis_uuid }}";
+
+    {% if REFINERY_REPOSITORY_MODE %}
+      var REFINERY_REPOSITORY_MODE = true;
+    {% else %}
+      var REFINERY_REPOSITORY_MODE = false;
+    {% endif %}
+
+    {% if user.is_authenticated %}
+      var REFINERY_USER_AUTHENTICATED = true;
+    {% else %}
+      var REFINERY_USER_AUTHENTICATED = false;
+    {% endif %}
+
+    var csrf_token = "{{ csrf_token }}";
+  </script>
+
+  <!-- Script for popup content in the tool status and analyses icon-->
+  <script type="text/ng-template" id="nodeselectionpopover.html">
+    <rp-node-selection-popover-detail>
+    </rp-node-selection-popover-detail>
+  </script>
+
+  <script type="text/javascript" src="{% static "js/refinery/solr/solr_utilities.js" %}"></script>
```

Comment:
These libraries and scripts below need to be reviewed. I'll put it on my agenda before leaving.